src/ProjectTemplates/test/ItemTemplateTests/RazorComponentsTests.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Threading.Tasks;
+using Templates.Test.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Templates.Items.Test
+{
+    public class RazorComponentsTest
+    {
+        public RazorComponentsTest(ProjectFactoryFixture projectFactory, ITestOutputHelper output)
+        {
+            ProjectFactory = projectFactory;
+            Output = output;
+        }
+
+        public Project Project { get; set; }
+
+        public ProjectFactoryFixture ProjectFactory { get; }
+        public ITestOutputHelper Output { get; }
+
+        [Fact]
+        public async Task RazorComponentsItemTemplate()
+        {
+            Project = await ProjectFactory.GetOrCreateProject("razorcomponentitem", Output);
+
+            var createResult = await Project.RunDotNetNewAsync("razorcomponent --name Steve");
```

Comment:
I'm gonna embody steve and say don't use team members names on tests please :).

Reply:
"Steve" has been my default name for things since forever, but whatever, I'll just a non-name.